### PR TITLE
SDK-245 [Python SDK] Add non hashing mode for key1-key10

### PR DESCRIPTION
**Pull request by RommeroInCountry**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-245](https://incountry.atlassian.net/browse/SDK-245)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by VladimirLevitskiy — APPROVED** [review]
LGTM

**Review by VladimirLevitskiy — DISMISSED** [review]

File: tests/unit/test_crypto_utils.py [hunk 1]
```diff
@@ -57,3 +57,197 @@ def test_is_json(data, result):
...
+@pytest.mark.happy_path
+def test_hash_object_record_keys_without_search_keys(
+    key_value_before,
+    normalize,
+    key_value_after,
+    non_search_keys_data_with_normalize,
+    non_search_keys_data_without_normalize,
+):
+    for key in SEARCH_KEYS:
+        print(key, normalize, key_value_before, key_value_after)
```
Comment (artemgolozubov) [inline tests/unit/test_crypto_utils.py:138]: Please remove this print() call

**Review by artemgolozubov — CHANGES_REQUESTED** [review]
The rest LGTM

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by VladimirLevitskiy — APPROVED** [review]

**Pull request merged**